include/net/gptp.h

```diff
@@ -165,17 +165,10 @@ struct gptp_hdr {
 	s8_t log_msg_interval;
 } __packed;
 
-/*
- * TODO: k_uptime_get need to be replaced by the MAC ptp_clock.
- * The ptp_clock access infrastructure is not ready yet
- * so use it for the time being.
- * k_uptime time precision is in ms.
- */
-#define GPTP_GET_CURRENT_TIME_NANOSECOND() (k_uptime_get() * 1000000)
-#define GPTP_GET_CURRENT_TIME_USCALED_NS(uscaled_ns_ptr)		\
+#define GPTP_GET_CURRENT_TIME_USCALED_NS(port, uscaled_ns_ptr)		\
```

Comment:
Though wait, what is "uscaled" in "USCALED_NS"?

Reply:
From IEEE 802.1AS spec: "The UScaledNs type represents unsigned values of time and time interval in units of 2 – 16 ns."

Reply:
Thanks